docs/getting_started-new/index.md

```diff
@@ -1,7 +1,180 @@
+## Install Taipy with pip
+
+To get started with Taipy, ensure that you have Python (version 3.8 or higher) and
+[pip](https://pip.pypa.io) installed on your system. Once you have these prerequisites, 
+open your terminal or command prompt and run the following command to download and
+install the latest stable release of Taipy.
+
+``` console
+$ pip install taipy
+```
+
+!!! info "Other Installation Options"
+    
+    If you need alternative installation methods or if you don't have Python 
+    or pip installed, please refer to the [installation page](installation/index.md)
+    for additional instructions.
+
+
+## First Taipy Scenario 
+Taipy introduces the concept of a *Scenario* to model pipeline executions. 
+A *Scenario* consists of an execution graph, often represented as a Directed 
+Acyclic Graph (DAG), where tasks (or functions) exchange data. Your scenario 
+can be as simple or as complex as your needs require.
+
+For your first scenario, we'll create a simple "Hello World" example 
+to demonstrate the configuration process. Here's an illustration of the 
+scenario's execution graph made of two data nodes (blue boxes) and one task 
+(orange box):
+
+![hello world example](hello_world.svg){width=50%}
+
+It consists in one input data node named *name*. Then a task named 
+*build_message* takes the first data node and returns a second 
+data node named *message*.
+
+Building the corresponding Taipy application requires three easy steps.
+
+### Configuring the application
+
+The first step involves configuring the execution graph. This includes 
+configuring data nodes, tasks, and scenario itself. 
+Use the following Python code:
+
+```python linenums="1"
+from taipy import Config
+
+
+def build_message(name: str):
+    return f"Hello {name}!"
+
+
+name_data_node_cfg = Config.configure_data_node(id="input_name")
+message_data_node_cfg = Config.configure_data_node(id="message")
+build_msg_task_cfg = Config.configure_task("build_msg", build_message, name_data_node_cfg, message_data_node_cfg)
+scenario_cfg = Config.configure_scenario("scenario", task_configs=[build_msg_task_cfg])
+```
+
+- Lines 4-5 define the function that the task will use during execution.
+- Lines 8-9 configure the two data nodes, *input_name* and *message*.
+- Line 10 configures a task named *build_msg* associated with the `build_message()`
+  function, specifying input and output data nodes.
+- Finally, line 11 configures the execution graph of the scenario providing 
+  the previously configured task.
+
+### Running The Core service
+
+Running the Core service allows Taipy to process the configuration made in the 
+previous step and set up the scenario management feature.
+
+Include the following code in your Python script:
+
+``` python linenums="1"
```

Comment:
The convention you use basically everywhere is not to have this white space after the three backticks